tests/sidetrail/README.md

```diff
@@ -129,12 +129,10 @@ export S2N=<path_to_s2n>
 
 ### Building the image
 
-To build the image, start with `s2n/codebuild/spec/sidetrail/Dockerfile`
-
-```shell
-cd $S2N
-docker build -f codebuild/spec/sidetrail/Dockerfile --tag sidetrail .
-```
+To build the image, run this scripts:
```

Comment:
Hey James!
Nope ,I didn't run the scripts on my system and I have no idea how would those behave on any platform.

About the second point, it would be great. Let me work on it!